deepchem/utils/electron_sampler.py

```diff
@@ -206,7 +210,15 @@ def move(self,
                                           size=np.shape(self.x)[0]))
                     cond = move_prob < ratio
                     lp1 = np.where(cond, lp2, lp1)
-                    self.x = np.where(cond[:, None, None, None], x2, self.x)
+                    tmp_sampled = np.where(cond[:, None, None, None], x2,

```

Comment:
Should we factor this out into a helper function? It's repeated 4 times in this code